README.md

```diff
@@ -54,6 +54,7 @@ The following settings are stored in `template_config.yml`.
                         org (optional): github organization
                         revision (optional): the git commit hash to install.
                                              "Required PR" in a commit message will override this.
+                        pytest_pyargs (optional): string to by passed to pytest via pyargs
```

Comment:
I'd expect a list of plugins from which we also run the functional tests.

Reply:
Ah, I misread. The context in the diff was not enough without expanding.

Reply:
But also to me that sounds like a feature switch:
`--pyargs              try to interpret all arguments as python packages.`

So technically we only add arguments to pytest, and the switch tells pytest to interpret _everything_ as python modules.

Reply:
This is better I think because it doesn't rely on the assumption that all tests are in a canonical place. Also it's simpler because I don't have to build that string up in code, the data can just have it.

Reply:
I agree (see my second comment) i just don't really dig the name. Is it `test_module`?

Reply:
What i like about this approach is that you could specify a subset of tests to run with pulpcore. But i'd rather have a `@pytest.mark.run_with_core` for this to add visibility.

Reply:
I named it with `pyargs` because it's possible we may want to add a section that doesn't use `pyargs` perhaps? Like passing the string directly to pytest.